src/test/regress/output/multi_complex_count_distinct.source

```diff
@@ -142,6 +274,45 @@ SELECT
       12417 |     3
 (10 rows)
 
+-- group by on non-partition column
```

Comment:
it would be useful to have also count(distinct)'s without a filter on the same query such as;


```
SELECT                                                                                                                                                                                                                                                                 l_suppkey, count(DISTINCT l_partkey) FILTER (WHERE l_shipmode = 'AIR'), count(distinct l_partkey), count(distinct l_shipdate)                                                                                                                                                 FROM lineitem                                                                                                                                                                                                                                                                   GROUP BY l_suppkey                                                                                                                                                                                                                                                              ORDER BY 2 DESC, 1 DESC                                                                                                                                                                                                                                                         LIMIT 10;
```

Reply:
added